src/components/MessageCard/MessageCard.jsx

```diff
@@ -97,14 +98,15 @@ export const MessageCard = React.memo(
             <MessageCardSubtitle subtitle={subtitle} />
             <MessageCardContent
               withMargin={!!(title || subtitle)}
-              render={!!(body || image || children)}
+              render={!!(body || image || children || video)}
             >
               <MessageCardBody
                 body={body}
                 onClick={onBodyClick}
                 variables={variables}
               />
               <MessageCardImage image={image} onLoad={makeMessageVisible} />
+              <MessageCardVideo video={video} onLoad={makeMessageVisible} />
```

Comment:
Suggestion, I think is clearer that MessageCardVideo will only render if `video` is present if you put it here:
```jsx
{ video ? <MessageCardVideo video={video} onLoad={makeMessageVisible} /> : null }
```

remove this check in `MessageCardVideo`: 
```js
if (!video) {
  return null
}
```
And make the `video` prop in there required.

Reply:
Will do! (if there isn't any reason why I did it like this, but I'll take a look)